static/scss/answers/interactive-map/VerticalFullPageMap.scss

```diff
@@ -562,9 +562,8 @@
     }
```

Comment:
would something like changing `.HitchhikerLocationStandard-distance` to be 
```css
position: absolute;
right: 0px;
```
 fix the card body being lower?

Reply:
^there might be some weirdness with long distances paired with long titles

Reply:
Ya, that would fix the card body being lower, however that still makes it susceptible to being overlapped by long titles or lots of text in the card body

Reply:
ah got it